coverage.md

```diff
@@ -26,6 +26,7 @@ This file contains list of features and their test coverage.
 | As a user, I can re-sync custom repository metadata when it was the only change in a repository | YES |  |
 | As a user, I can sync an existing advisory whose dates are timestamps (as opposed to datetimes) | NO  | Example: https://updates.suse.com/SUSE/Updates/SLE-SERVER/12-SP5/x86_64/update/ |
 | As a user, I can sync repos with repomd.xml files whose filenames contain dashes (e.g., app-icons.tar.gz) | NO  | Example: https://updates.suse.com/SUSE/Updates/SLE-SERVER/12-SP5/x86_64/update/ |
+| As a user, I can see Modulemd's `description` and `profiles` fields. | YES | |
 | As a user, the content metadata being saved to Pulp is correct | PART | Packages and distribution trees are covered. Updateinfo and Groups metadata is not covered. "Weird" cases not covered. |
```

Comment:
@pavelpicka What I meant is, it should be covered by `As a user, the content metadata being saved to Pulp is correct`